packages/search/src/features/registration/birth/service.ts

```diff
@@ -36,32 +36,32 @@ import { getSubmittedIdentifier } from '@search/features/search/utils'
 import {
   getComposition,
   SavedComposition,
-  SavedBundle,
   Patient,
   getFromBundleById,
   getTaskFromSavedBundle,
   resourceIdentifierToUUID,
   SavedRelatedPerson,
   findFirstTaskHistory,
-  ValidRecord,
+  SavedBundle,
   getBusinessStatus
 } from '@opencrvs/commons/types'
+import { findAssignment } from '@opencrvs/commons/assignment'
 
 const MOTHER_CODE = 'mother-details'
 const FATHER_CODE = 'father-details'
 const INFORMANT_CODE = 'informant-details'
 const CHILD_CODE = 'child-details'
 const BIRTH_ENCOUNTER_CODE = 'birth-encounter'
 
-export async function indexRecord(record: ValidRecord) {
+export async function indexRecord(record: SavedBundle) {
```

Comment:
Why are we reverting to the `SavedBundle` type?

Reply:
I was thinking that the indexed records may be in progress and incomplete, and might not be 'Valid Records' but now that I look into the ValidRecord type, it includes InProgressRecord's so maybe it makes more sense

Reply:
ValidRecord has SavedBundleEntries which requires a URLReference, which I don't think the indexRecord would always necessarily have, if it's indexed before Hearth
```

// /fhir/Patient/3bd79ffd-5bd7-489f-b0d2-3c6133d36e1e/_history/94d9feab-78f9-4de7-9b4b-a4bcbef04a57
export type URLReference = Nominal<string, 'URLReference'>
```

Reply:
hm don't we always index after saving to Hearth?